releasenotes/notes/add-consumer-grade-gpu-support-770dd9a77367c198.yml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add suppot for consumer grade nvidia gpus. Those cards expose
+    audio devices which id, needs to be added as one of kernel 
+    arguments in vfio-pci.id variable along with display_id to work properly.
+    Support for Nvidia p4000 is also added as working example of those cards.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)